packages/mobile/src/transactions/actions.ts

```diff
@@ -24,10 +24,8 @@ export enum Actions {
   REFRESH_RECENT_TX_RECIPIENTS = 'TRANSACTIONS/REFRESH_RECENT_TX_RECIPIENTS',
   UPDATE_RECENT_TX_RECIPIENT_CACHE = 'TRANSACTIONS/UPDATE_RECENT_TX_RECIPIENT_CACHE',
   UPDATE_TRANSACTIONS = 'TRANSACTIONS/UPDATE_TRANSACTIONS',
-  // Remove legacy actions once the multitoken support feature is fully released
+  // Remove legacy action once the multitoken support feature is fully released
   ADD_STANDBY_TRANSACTION_LEGACY = 'TRANSACTIONS/ADD_STANDBY_TRANSACTION_LEGACY',
-  REMOVE_STANDBY_TRANSACTION_LEGACY = 'TRANSACTIONS/REMOVE_STANDBY_TRANSACTION_LEGACY',
```

Comment:
This flag is not on yet, right? So we shouldn't remove this yet?

Reply:
Same as before, I merged both "remove logic" in one.